### DISCUSSION TITLE
Borderless Ore/Smooth Blocks

### DISCUSSION THREAD

**Discussion by AtlasOfEden** [BedrockTweaks/Files#370] [Ideas]
Remove the edge bevel from the Ore Blocks and polished/smooth blocks. Examples being Iron Block, Gold Block, Diamond Block, Polished Blackstone, Polished Deepslate, Polished Granite, etc.

**Comment by SquatchHunter** [comment]
example please?

also community packs will be done after addons are
we still dont have a timeline

**Reply by AtlasOfEden** [reply]
Like how you removed the border from glass, smooth off the bevel on the polished blocks so the textures becomes seamless. The bevel is the obvious line at the edge of those blocks.
